ergo-wallet/src/main/scala/org/ergoplatform/wallet/secrets/SecretStorage.scala

```diff
@@ -16,6 +20,9 @@ trait SecretStorage {
 
   def lock(): Unit
 
+  /**
```

Comment:
This is not related to SecretString, but since this class is touched, please add ScalaDocs for all methods.
A lot of quesitons can be addressed in docs:
- should secretFile be open, and how
- what are the implications of isLocked = true
- what means secret = None
- how state should change on unlock and lock

Reply:
Added docs, please review